spec/validator.js

```diff
@@ -50,4 +50,14 @@ describe('Validator constructor', function() {
   it('should have a check method', function() {
     expect(validator.check).to.be.a.function;
   });
+
+  it('should handle undefined data', function() {
+    var validator = new Validator(undefined, { name: 'required' })
+    validator.check();
```

Comment:
I don't think check is part of the public API. How about using something like this: http://chaijs.com/api/bdd/#method_throw to verify that the validator doesn't throw when fails() is called?